x-pack/plugins/security_solution/public/common/store/store.ts

```diff
@@ -176,7 +258,13 @@ export const createStore = (
   storage: Storage,
   additionalMiddleware?: Array<Middleware<{}, State, Dispatch<AppAction | Immutable<AppAction>>>>
 ): Store<State, Action> => {
-  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+  const enhancerOptions: EnhancerOptions = {
+    name: 'Kibana Security Solution',
+    actionSanitizer: actionSanitizer as EnhancerOptions['actionSanitizer'],
```

Comment:
Can we run those only in `dev`?

Reply:
that already happens, as the middleware is not even present outside of dev mode